auth/resource.hh

```diff
@@ -182,6 +194,23 @@ public:
 
 std::ostream& operator<<(std::ostream&, const service_level_resource_view&);
 
+///
+/// A "function" view of \ref resource.
```

Comment:
Is this "function" or "functions"? Or it's the same thing?

Reply:
The view may actually refer to either "function" or "functions" so maybe using both forms here isn't even wrong